includes/directory.h

```diff
@@ -0,0 +1,11 @@
+#ifndef DIRECTORY_H
+# define DIRECTORY_H
+
+#include "libft.h"
+#include <unistd.h>
+
+#define BUFFER_SIZE 255
```

Comment:
BUFFER_SIZE is too generic, it could be used by any other library. 
you can make it something like DIRECTORY_READ_BUFFER_SIZE
or maybe better is to put something like #define __DIRBUFSIZE in the .c file, so it doesnt colide with other libs